vllm_spyre/platform.py

```diff
@@ -168,10 +172,9 @@ def get_warmup_shapes(cls, scheduler_config) -> tuple[dict[str, int], ...]:
                     "The lists in VLLM_SPYRE_WARMUP_PROMPT_LENS and "
                     "VLLM_SPYRE_WARMUP_NEW_TOKENS must have equal length")
 
-        if not envs_spyre.VLLM_SPYRE_USE_CB:
```

Comment:
duplicated check, the function is called before checking `if not envs_spyre.VLLM_SPYRE_USE_CB`